src/pages/Dashboard/Families/FamilyDetail/FamilyInfo.tsx

```diff
@@ -0,0 +1,36 @@
+import type { MantineColor } from '@mantine/core';
+import { Badge, Box, Space, Text, Title } from '@mantine/core';
+
+interface Props {
+  title: string;
+  value: string;
+  isBadge?: boolean;
+  badgeStatus?: Extract<MantineColor, 'green' | 'red' | 'yellow'>;
+}
+
+export const FamilyInfo = ({ title, value, isBadge, badgeStatus }: Props) => {
```

Comment:
this naming is not very good
ussually when we use words like `Info` in naming it's a sign of missed concept 
and this case I think it is

Reply:
What ur suggestion then?